standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2765,7 +2822,8 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
 
   list<string> get_partition_names(1:string db_name, 2:string tbl_name, 3:i16 max_parts=-1)
                        throws(1:NoSuchObjectException o1, 2:MetaException o2)
-
+  list<string> get_partition_by_names_req(1:PartitionsRequest partitionReq)
```

Comment:
this method name is confusing, as it's not meant to get the partition, change it to get_partition_names(1:PartitionsRequest partitionReq)?